.travis.yml

```diff
@@ -67,6 +67,14 @@ jobs:
         - export ARCH=x86_64
         - bash kconfig/kconfig-sst.sh
         - KCFLAGS="-Wall -Werror" make -j`getconf _NPROCESSORS_ONLN`
+    - name: "BUILD GCC9 SOF Kernel i386"
+      script:
+        - export ARCH=i386 KCFLAGS="-Wall -Werror"
+        - bash kconfig/kconfig-sof-default.sh
+        - make modules_prepare
+        - make -j`getconf _NPROCESSORS_ONLN` sound/
+        - make -j`getconf _NPROCESSORS_ONLN` drivers/soundwire/
+        - make -j`getconf _NPROCESSORS_ONLN`
```

Comment:
Nit1: `cmd` is basically deprecated, prefer `$( )` https://github.com/koalaman/shellcheck/wiki/SC2006

Nit2: `nproc` seems more popular than `getconf _NPROCESSORS_ONLN`. It's shorter and can apparently return a lower value in some cases.

Reply:
@marc-hb do you mind sending a follow-up patch? I don't speak bash fluently.

Reply:
> do you mind sending a follow-up patch?

PR #2405